### PR TITLE
Update Requests version to 2.20.0 to handle security risk

### PR DESCRIPTION
**Pull request by kalyc**
## Description ##
Update Requests version to 2.20.0 to handle security risk

Fixes https://github.com/apache/incubator-mxnet/issues/13032

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [X] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Update Requests version to 2.20.0 in `setup.py`

### CODE REVIEW

**Comment by kalyc** [comment]
@nswamy @szha

**Comment by kalyc** [comment]
@mxnet-label-bot add [pr-awaiting-review]

**Comment by kalyc** [comment]
Tested on `Deep Learning Base AMI (Ubuntu) Version 10.0 (ami-050ffa288f9b282c2)`
using `make -j $(procs)` & `python setup.py install`
```
>> import mxnet as mx
```

File: python/setup.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 else:
     from setuptools import setup
     from setuptools.extension import Extension
-    kwargs = {'install_requires': ['numpy<=1.15.2,>=1.8.2', 'requests<2.19.0,>=2.18.4', 'graphviz<0.9.0,>=0.8.1'], 'zip_safe': False}
+    kwargs = {'install_requires': ['numpy<=1.15.2,>=1.8.2', 'requests>=2.20.0', 'graphviz<0.9.0,>=0.8.1'], 'zip_safe': False}
```
Comment (szha) [inline python/setup.py]: requests follow semver. add a limit to `,<3`

Comment (kalyc) [reply python/setup.py]: Done @szha

**Pull request merged**